src/resq.ts

```diff
@@ -1,75 +1,59 @@
-import { filterNodesBy, findSelectorInTree, buildNodeTree } from './utils'
+import { RESQNode } from '../types';
```

Comment:
I think it is better to use CamelCase even for abbreviations:
```suggestion
import { ResqNode } from '../types';
```